tasks.py.jinja

```diff
@@ -283,14 +288,16 @@ def lint(c, verbose=False):
 
 
 @task(develop)
-def start(c, detach=True, debugpy=False):
+def start(c, detach=True, debugpy=False, wait=False):
```

Comment:
I don't think you need this argument.

Doing `env SERVICES_WAIT_TIME=0 inv start` would be the same, after all...

Reply:
Changes made